docs/other/fontsobject.rst

```diff
@@ -79,6 +79,28 @@ Array of fontObjects; read-only.
 Methods
 =======
 
+.. _FontsObject.fontsWithDefaultDesignAxes:
+
+FontsObject.fontsWithDefaultDesignAxes()
```

Comment:
A big one here: this is an attribute, like allFonts, and not a method. Make sure this is reflected elsewhere as well.